src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck starts work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stops work");

```

Comment:
Similarly, the stopWork method message 'Truck stops work' could be more descriptive. Consider specifying the type of work being stopped, such as 'Truck ends its delivery route' or 'Truck concludes transportation work'.